include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -153,12 +153,15 @@ struct base_turn_handler
             typename UmbrellaStrategy,
             typename TurnInfo
     >
-    static inline void both_collinear(UniqueSubRange1 const& range_p,
+    static inline void both_collinear(
+            UniqueSubRange1 const& range_p,
             UniqueSubRange2 const& range_q,
-            UmbrellaStrategy const& umbrella_strategy,
+            UmbrellaStrategy const&,
             std::size_t index_p, std::size_t index_q,
             TurnInfo& ti)
     {
+        boost::ignore_unused(range_p);
+        boost::ignore_unused(range_q);
```

Comment:
Btw, you could pass multiple arguments so this could be one line, but it's ok as it is.